config.py

```diff
@@ -109,16 +109,17 @@ def StripFreeParams(point):
         'near_run3',
         'cmd':
         '''/bin/bash -l -c 'source /opt/FairShipRun/config.sh; '''
-        '''python2 /code/slave.py '''
+        '''python2 /code/weighter.py '''
```

Comment:
I'd appreciate if you didn't change the general template. You could either modify it in the submission script or add a second template. I think I'd go with the first option.

Otherwise all the optimisation scripts will break.

Reply:
I just thought that we could use separate branches. But okay, I'll create another template.

Reply:
I think maintaining two separate branches for this would get hard to maintain. I especially would really like to avoid having more than one good docker image at a time.

But instead of creating a separate template, it's probably easiest if you just modify the necessary fields in the script that submits your docker jobs, e.g.
```python3
template['cmd']='''/bin/bash -l -c 'source /opt/FairShipRun/config.sh; '''
'''python2 /code/weighter.py ... '''
```

Reply:
Okay, I agree!